app/perfsymboltable.cpp

```diff
@@ -300,10 +301,11 @@ QFileInfo PerfSymbolTable::findFile(const char *path, const QString &fileName,
     return fullPath;
 }
 
-// returns true for special regions, such as [heap], [vdso], [stack], ... as well as //anon
+// returns true for special regions, such as [heap], [vdso], [stack], [kernel.kallsyms]_text ... as well as //anon
 static bool isSpecialRegion(const QByteArray& filePath)
 {
-    return (filePath.startsWith('[') && filePath.endsWith(']'))
```

Comment:
simplify:

```
return filePath.startsWith('[') || filePath == "//anon" || ...
```

I don't see why it would matter that we get the closing bracket

Reply:
Someone might map /usr/bin/[... I'll simplify though.

Reply:
Done.